api4/team.go

```diff
@@ -20,6 +20,7 @@ func InitTeam() {
 
 	BaseRoutes.Team.Handle("", ApiSessionRequired(getTeam)).Methods("GET")
 	BaseRoutes.Team.Handle("/stats", ApiHandler(getTeamStats)).Methods("GET")
+	BaseRoutes.Team.Handle("/members", ApiHandler(getTeamMembers)).Methods("GET")
```

Comment:
Noted. I've updated route to session required, and test from forbidden to unauthorized.